### PR TITLE
Create sponsor component and button component for first byte

### PR DESCRIPTION
**Pull request by Davooood90**
- Created a folder looking custom component to display sponsor(s)
- Created a 8-bit style button for redirects

what all the components look like (not arranged properly yet):
Screenshot 2024-10-16 at 9 36 45 PM

### CODE REVIEW

**Comment by sonatuna** [comment]
looks great! was the absolute position just for display purposes?

**Review by sonatuna — APPROVED** [review]

File: src/pages/first-byte-page/FirstBytePage.tsx [hunk 1]
```diff
@@ -15,8 +17,14 @@ const FirstBytePage: FC = () => {
         styles={{ width: "600px", height: "400px" }}
       >
         <p>This window has custom width and height.</p>
-        <button>Click Me</button>
       </Window>
+      <div className="fb-sponsors">
+        <Folder
+          sponsorName="mastercard"
+          sponsorLogo="https://media.discordapp.net/attachments/859197594831552542/1296256765914120192/image.png?ex=6711a0b0&is=67104f30&hm=756467d14d2fbabd2494a0f4cfa8769cad9f67e8db7dea38c71c62976bed9991&=&format=webp&quality=lossless&width=712&height=471"
+        ></Folder>
```
Comment (LeoShangTang) [inline src/pages/first-byte-page/FirstBytePage.tsx:25]: If I'm not wrong, I think we have the image of master card in our repo. I think we can maybe have the backup as the master card in our repo??? 

But it's not too big of a deal

**Review by LeoShangTang — APPROVED** [review]
I think it looks good

The css for the folder is scary though so I aint touching that until a bug shows up LOL

Only sus part is the absolute positioning, so I'll take a look at them, but I think this ticket can get merged

**Comment by Davooood90** [comment]
Ermmm yeah the css for the folder can cause quite a headache. I wasn't too sure how to get the effect of the folders stacking on each other so I framed the two folders in a div and then made each of the folders absolute so that they would stack. I'll merge it for now, but we'll prob have to change it so that scaling on different screen sizes will work better.

**Pull request merged**